packages/core/src/services/aws-saml-assertion-extraction-service.ts

```diff
@@ -14,10 +14,14 @@ const authenticationUrlRegexes = new Map([
       /^https:\/\/login\.okta\.com\/.*/,
       /^https:\/\/accounts\.google\.com\/ServiceLogin.*/,
       /^https:\/\/login\.microsoftonline\.com\/.*\/oauth2\/authorize.*/,
+      /^https:\/\/account\.activedirectory\.windowsazure\.com\/applications\/signin\/*?tenantId=*/,
     ],
   ],
 ]);
-const samlAssertionRegexes = new Map([[CloudProviderType.aws, [/^https:\/\/signin\.aws\.amazon\.com\/saml/]]]);
```

Comment:
I would suggest swapping this one as well to have the public AWS Cloud first and GovCloud Second